src/app/public/modules/popover/popover-content.component.ts

```diff
@@ -215,6 +218,10 @@ export class SkyPopoverContentComponent implements OnInit, OnDestroy {
     this.horizontalAlignment = config.horizontalAlignment;
     this.placement = config.placement;
     this.popoverTitle = config.popoverTitle;
+    /* istanbul ignore else */
+    if (config.popoverType) {
```

Comment:
This is somewhat minor - but I would personally prefer that we test this. We do something similar here with placement on line 127 of the directive test (which is where all the popover tests live) where we give different values and make sure that the `ng-class` is working on the content.

Could we just add one test that tests that the class that we add is right for when it is `info`, `danger`, and when it is undefined?